src/Build/Xml/XmlReaderExtension.cs

```diff
@@ -26,6 +28,9 @@ internal static XmlReaderExtension Create(string filePath, bool loadAsReadOnly)
         private readonly Stream _stream;
         private readonly StreamReader _streamReader;
 
+        private static volatile PropertyInfo _normalizationPropertyInfo;
```

Comment:
Whenever one uses `volatile` one should add a comment explaining what is going on ;)

Reply:
I wonder if explicitly calling `Volatile.Read` and `Volatile.Write` would make it more obvious. Comment added!